flips/20211007-transaction-fees.md

```diff
@@ -0,0 +1,264 @@
+
+Each part of the fees can be further split into *effort*(<img src="https://render.githubusercontent.com/render/math?math=E">) and an *effort cost factor*(<img src="https://render.githubusercontent.com/render/math?math=f">) of FLOW per unit of effort. The *effort* required for a specific transaction should be independent of when the transaction is sent[^1] and should only depend on the properties of the transaction. On the other hand the *effort cost factor* would change over time depending on network activity.
+
+[^1]: Assuming there weren't any changes to the state that would make the transactions script execute differently.
+
+The final result in equation form looks like this:
+
+<img src="https://render.githubusercontent.com/render/math?math=F%5E%7Bmax%7D%20%3D%20f%20(%20E_i%20%2B%20E_e%20)">
+
+<!-- TODO update diagram -->
+
+![Alt text](20211007-transaction-fees/fees_diagram.svg)
+
+- <img src="https://render.githubusercontent.com/render/math?math=F_i">: Inclusion fees; Represent the cost of including the transaction in a collection.
+    - <img src="https://render.githubusercontent.com/render/math?math=f">: The current cost of one unit of effort.
+    - <img src="https://render.githubusercontent.com/render/math?math=E_i">: Inclusion effort; The time invariant effort required to handle this transaction (include it into a collection, check signatures are valid, check that the payer can pay, transfer the transaction over the wire,...). The inclusion effort is known before the transaction is executed.
+    - <img src="https://render.githubusercontent.com/render/math?math=E_c">: Execution effort; The time invariant effort required to execute this transaction (run the script, process the results, send results to verification node, verify the results, ...). The Execution effort cannot be know without executing the transactions script.
+
+
+### Fee effort cost factor
+
+The effort cost factor <img src="https://render.githubusercontent.com/render/math?math=f"> should be defined on a smart contract and adjustable via the service account admin resource and should be accessible for everyone to read. 
+
+The effort cost factor can be used in three different ways:
+
+1. The main usage of the effort cost factors is adjusting it when the network is under load. 
+
+2. The second reason to tweak the effort cost factors is that over time as hardware improves, the same amount of effort will be easier to handle by the nodes.
+
+3. A potential third use of the effort cost factor would be to keep the cost of transaction stable according to the USD value of flow. This would require a periodic job to run, check the USD value of FLOW and update the cost factor to account for any changes. This would allow for the average transaction fees to stay relatively close to a set USD value.
```

Comment:
Can we update to “fiat” value? Im trying to get us away from overfitting to US market